certified-connectors/LCP - iCordis/apiProperties.json

```diff
@@ -0,0 +1,37 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "Export",
```

Comment:
@stijn-lcp can you please confirm if this is a real value? if yes then please replace with a dummy value. You can use something like "Please enter your client id".